src/chapters/introduction/using-this-book.rst

```diff
@@ -1,2 +1,31 @@
 Using This Book
 ===============
+
+Throughout this book, you will find a variety of different sections and practice exercises.
+We are writing this guide to help you make the most of the book.
+
+Concept checks
+--------------
+
+Many sections end with a "Check Your Understanding".
+This section is full of questions for you to double check that you understand the concepts in the reading.
+Your score does not count towards your final grade in the class
```

Comment:
Punctuation/phasing edit: "Although your score does not count towards your final grade in the class, you should use it to help evaluate your understanding of the main concepts."

Reply:
Thanks!